holidays/financial/ny_stock_exchange.py

```
@@ -285,3 +655,126 @@ class NewYorkStockExchangeStaticHolidays:
+    # Partial suspensions of work during the day are also not implemented:
+    # * May 20, 1910 (Fri) - Closed from 10:00 am to 12:00 noon. Funeral of King Edward VII.
+    # * Apr. 14, 1913 (Mon) - Closed from 10:00 am to 12:00 noon. Funeral of J. P. Morgan.
+    # * Sept. 22, 1913 (Mon) - Closed from 10:00 am to 12:00 noon. Funeral of
+    #   Mayor William J. Gaynor.
+    # * July 11, 1918 (Thu) - Closed from 11:00 am to 12:00 noon. Funeral of former Mayor
+    #   John Purroy Mitchell.
+    # * Aug. 8, 1923 (Wed) - Closed from 11:00 am to 12:30 pm during funeral services for President
+    #   Warren G. Harding in Washington, DC.
+    # * Jan. 28, 1936 (Tue) - Closed from 10:00 to 11:00 am. Funeral of King George V of England.
+    # * May 18, 1942 (Mon) - Closed from 12:00 noon to 1:00 pm. NYSE 150th anniversary.
+    # * June 19, 1945 (Tue) - Closed from 11:00 am to 1:00 pm – Parade for General Eisenhower.
```

Ruff (0.14.7)

Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)